contrib/multivariate-tada/genunivdata.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import torch
+
+#Simulates univariate data based on demo mutation rates in original TADA code
+
+def genUnivData(gamma, q, N_family, N_case, N_control, mut_1, mut_2):
+    #Rates for poisson distributions that the counts follow
+
+    X_d1_rates = 2 * mut_1 * gamma * N_family
+    X_d2_rates = 2 * mut_2 * gamma * N_family
+    X_ctrl_rates = 5000 * [q * N_control]
+    X_case_rates = 5000 * [q * gamma * N_case]
+    X_t_rates = 5000 * [q * gamma * N_family]
+    X_nt_rates = 5000 * [q * N_family]
+
+    #Stacks poisson rates and samples
+    cls1 = torch.stack([X_d1_rates,torch.tensor(X_t_rates),torch.tensor(X_nt_rates),torch.tensor(X_case_rates),torch.tensor(X_ctrl_rates)])
```

Comment:
what's happening here?  AFAICT this will fail unless `X_d1_rates` is a tensor,  which means that at least one of the quantities used to define it must also be a tensor.  Type annotations would help to clarify this for the reader